src/map/elemental.c

```diff
@@ -97,13 +97,13 @@ int elemental_create(struct map_session_data *sd, int class_, unsigned int lifet
 	summon_level = db->status.size+1; // summon level
 
 	//[(Caster's Max HP/ 3 ) + (Caster's INT x 10 )+ (Caster's Job Level x 20 )] x [(Elemental Summon Level + 2) / 3]
-	ele.hp = ele.max_hp = (sd->battle_status.max_hp/3 + sd->battle_status.int_*10 + sd->status.job_level) * ((summon_level + 2) / 3);
+	ele.hp = ele.max_hp = ((sd->battle_status.max_hp / 3) + (sd->battle_status.int_ * 10) + (sd->status.job_level * 20)) * ((summon_level + 2) / 3);
```

Comment:
@shennetsind  Committed this line in c728c1f7. And the comment/formula hasn't been updated since. Haru edited in commit b791b665 without updating the formula or comment. It would be appreciated if you added the source of your information in the PR report so a reviewer would know that the formula is correct.